compiler/damlc/daml-stdlib-src/DA/Internal/Template/Functions.daml

```diff
@@ -243,6 +241,7 @@ instance Eq AnyChoice where
 instance Eq AnyContractKey where
```

Comment:
:tada: Now I can add `Ord` instances :slightly_smiling_face:

Reply:
Yes :)

Reply:
The order for enums and variants still needs to be adjusted.